pkg/pool-hooks/contracts/ECLPSurgeHook.sol

```diff
@@ -53,6 +60,14 @@ contract ECLPSurgeHook is SurgeHookCommon {
         TokenConfig[] memory tokenConfig,
         LiquidityManagement calldata liquidityManagement
     ) public override onlyVault returns (bool success) {
+        (IGyroECLPPool.EclpParams memory eclpParams, ) = IGyroECLPPool(pool).getECLPParams();
+
+        // The surge hook only works for pools with a rotation angle between 30 and 60 degrees. Outside of this range,
+        // the computation of the peak price cannot be approximated by sine/cosine.
```

Comment:
I guess it's kind of obvious from context, but we could note that sin 30 degrees = cos 60 degrees = 0.5; sine monotonically increases and cosine monotonically decreases (from 0-90); so they're only *both* below 0.5 between 30 and 60. Above 60, sine will be > 0.5, and below 30, cosine will be > 0.5.